### ISSUE TITLE
Add Gnome support

### ISSUE THREAD

**Issue by BjarniRunar** [mailpile/gui-o-matic#3]
We would like to support the Gnome desktop. Some of the work is already done in the GTK base, but native indicator icons would be nice, as well as dock integration once that is in place.

It's possible that the appindicator module used in our Unity support does this already (potentially making this a no-op), but that needs to be tested and verified.
